Tests/Functional/Controller/TaskControllerTest.php

```diff
@@ -270,17 +270,17 @@ public function testPost(
         $locale = 'de'
     ) {
         $date = new \DateTime($schedule);
-        $scheduleDate = $date->format('c');
 
         $client = $this->createAuthenticatedClient();
         $client->request(
             'POST',
             '/api/tasks',
             [
                 'handlerClass' => $handlerClass,
-                'schedule' => $scheduleDate,
-                'entityClass' => $entityClass,
-                'entityId' => $entityId,
+                'date' => $date->format('Y-m-d'),
+                'time' => $date->format('H:i:s'),
+                'entity-class' => $entityClass,
```

Comment:
out of curiosity - is there a reason for using kebab case here? not sure if we do this anywhere else